src/fileio/fileio_party.cpp

```diff
@@ -30,6 +30,28 @@ extern cCustomGraphics spec_scen_g;
 static bool load_party_v1(fs::path file_to_load, cUniverse& univ, bool town_restore, bool in_scen, bool maps_there, bool must_port);
 static bool load_party_v2(fs::path file_to_load, cUniverse& univ);
 
+fs::path nav_get_party();
```

Comment:
If you must declare these two functions here in this file instead of including a header, please make the following two changes:

1. Move them both to the top of the file (that is, the second one should be immediately below this one instead of further down).
2. Declare them extern.